cmake/external_tbb_prebuilt.cmake

```diff
@@ -16,12 +16,15 @@
 
 include(ExternalProject)
 
-set(ARCHIVE_FILE_BASE tbb2019_20181203oss)
+
+set(NGRAPH_TBB_VERSION "2019_U3")
```

Comment:
Since this is used in two places (this file and external_tbb.cmake) can we move it up one level so it is only set in one place? The two places will get out of sync and lead to unexpected behavior

Reply:
Can we do this in a separate PR that maybe switches to just using the pre-built tbb, i.e. eliminating the second place?